proof/refine/ARM_HYP/Retype_R.thy

```diff
@@ -4154,21 +4078,21 @@ lemma createNewCaps_idle'[wp]:
              split del: if_split)
   apply (cases ty, simp_all add: Arch_createNewCaps_def
                       split del: if_split)
-         apply (rename_tac apiobject_type)
-         apply (case_tac apiobject_type, simp_all split del: if_split)[1]
-             apply wpsimp
-            (* The following step does not use wpsimp to avoid clarsimp_no_cond, which for some reason
-               leads to a failed proof state. If this could be fixed then the inclusion of
-               classic_wp_pre could also be removed. *)
-            including classic_wp_pre
-            apply (wp mapM_x_wp' createObjects_idle' threadSet_idle'
-                   | simp add: projectKO_opt_tcb projectKO_opt_cte
-                               makeObject_cte makeObject_tcb archObjSize_def
-                               tcb_cte_cases_def objBitsKO_def APIType_capBits_def
-                               vspace_bits_defs objBits_def
-                               createObjects_def tcb_cte_cases_neqs
-                   | intro conjI impI
-                   | clarsimp simp: curDomain_def)+
+        apply (rename_tac apiobject_type)
+        apply (case_tac apiobject_type, simp_all split del: if_split)[1]
+            apply wpsimp
+           (* The following step does not use wpsimp to avoid clarsimp_no_cond, which for some reason
+              leads to a failed proof state. If this could be fixed then the inclusion of
+              classic_wp_pre could also be removed. *)
```

Comment:
uh, do we need an issue for this?

Reply:
Maybe but it's not new and I'm not sure what I would put in that issue beyond what's in this comment. Happy to do that if you want.